docs/query-language.md

```diff
@@ -451,6 +451,13 @@ Returns `{{"attrs":{},"bits":[10]}`
 
 * bits are repositories which had at least 100 commits in the last year.
 
+Similar syntax is supported for less-than (LT or `<`), less-than-or-equal (LTE or `<=`), and greater-than-or-equal (GTE or `>=`). An interval with both bounds can be specified with the "BETWEEN" operator `><`, and a two-element list containing the lower and upper bounds of the interval:
+
+```
+Range(frame="stats", commitactivity >< [100, 200])
+```
+
+This is conceptually equivalent to the interval 100 < commitactivity < 200, but this chained comparison syntax is not currently supported. BETWEEN queries syntax is restricted to greater-than and less-than, but any valid interval on the integers can be represented this way.
```

Comment:
The `BETWEEN` query is actually inclusive, so its equivalent would be `100 <= commitactivity <= 200`.